File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    visited = {};
    for (const i in nums) {
        if (!visited[nums[i]]) {
            visited[nums[i]] = 1;
        } else {
        return true;
        };
    };

    return false;
};

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """

        visited = {};
        for i, num in enumerate(nums):
            if num not in visited:
                visited[num] = 1
            else:
                return True
        
        return False